inc/cleantalk-public-integrations.php

```diff
@@ -1449,7 +1449,7 @@ function ct_registration_errors($errors, $sanitized_user_login = null, $user_ema
         true
     );
     $ct_result        = $base_call_result['ct_result'];
-
+    ct_hash($ct_result->id);
```

Comment:
why call ct_hash if its results are not used further?

Reply:
Using a static variable in the function.